.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
Why do we need this ?

Reply:
It seems that it is created by default. I will delete it while committing